file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs

```diff
@@ -271,7 +274,8 @@ protected internal override async Task OnExecuteCommand()
                                         {
                                             logger.LogInformation("File Share Service upload files completed for file:{file} and BatchId:{BatchId} .", f.file.Name ,batchHandle.BatchId);
                                         }
-                                    }).ContinueWith(_ => openRead.Dispose());                            
+                                    },
+                                    newBatchFilesViewModel.Attributes.ToArray()).ContinueWith(t => { if (t.Exception != null) { throw t.Exception; } openRead.Dispose(); });                            
```

Comment:
Please ensure `openRead` is always disposed

Reply:
Removing this exception block as we are handling API exception at dot net client and from there will be returning DTO.
And here dispose of openRead is in place.
Thanks